docs/docs-content/clusters/cluster-management/backup-restore/enable-etcd-backup.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_label: "Configure etcd Backup"
+title: "Configure etcd Backup"
+description: "Learn how to enable scheduled etcd backups."
+hide_table_of_contents: false
+sidebar_position: 45
+tags: ["clusters", "cluster management", "backup"]
+---
+
+etcd backups are enabled by default and backups are performed on each etcd node. You can adjust the backup frequency,
+the location of the backup files, and the maximum number of backup files to retain by editing the YAML file of your
+cluster profile's Kubernetes layer.
+
+## Prerequisite
+
+- An active cluster in Palette.
+
+## Configure etcd Backup
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. From the **Main Menu**, select **Profiles**.
+
+3. Select the profile you use to deploy clusters. And then, select the Kubernetes layer of your profile.
+
+4. Edit the YAML file of your Kubernetes layer and add the following configurations. Depending on your Kubernetes
+   distribution, different configuration parameters are available.
+
+    <Tabs group="distribution">
```

Comment:
Lenny, do you know by chance if Microk8s also has this ability?

Reply:
I asked boobalan, and he said that he isn't sure about microk8s. I also see that microk8s uses dqlite3 instead of etcd for data store by default so this might not apply at all. I will ask akhilesh as well to make sure